Fixed mobile menu to top of screen

craigsteel opened the pull request.

Commit:
Fixed mobile menu to top of screen

cmccrorie commented:
@craigsteel 
Hi Craig Just looked at the pull request and tested it. I have found some issues with this.
 The first one is that when clicking on the "About" "Services" and "Exist-DB" tabs It does not respond unless I go to the "Careers" or "Contact" tab / page first then go back it will then work.  
 Seen in below screen shot. 

![Image from iOS](https://user-images.githubusercontent.com/68058507/110330304-880aa680-8015-11eb-933c-21b2cd0acfd6.png)


The second issue I found was that you can see the tabs from the main pages along the side of the menu screen. This is also followed by some text appearing at the bottom of the screen occasionally - this almost makes the screen look like it is glitching. Seen in the Screen shot below.

![Image from iOS (1)](https://user-images.githubusercontent.com/68058507/110330356-935dd200-8015-11eb-949b-8a53cabedac6.png)


Lastly, The menu icon covers up parts of the text when scrolling down pages which can become annoying to users of the site. 
Seen in the screen shot below.

![Image from iOS (2)](https://user-images.githubusercontent.com/68058507/110330387-99ec4980-8015-11eb-965a-c3ebbdf5cf4e.png)

craigsteel commented:
Hi, 
Really sorry about that. I’m just about finished re doing the mobile menu and some cleaning up of the css. will upload tonight.

regards Craig

> On 8 Mar 2021, at 13:51, cmccrorie <notifications@github.com> wrote:
> 
> 
> @craigsteel <https://github.com/craigsteel>
> Hi Craig Just looked at the pull request and tested it. I have found some issues with this.
> The first one is that when clicking on the "About" "Services" and "Exist-DB" tabs It does not respond unless I go to the "Careers" or "Contact" tab / page first then go back it will then work.
> Seen in below screen shot.
> 
>  <https://camo.githubusercontent.com/1cf69ce10f059160c65946941d3c913d0fdb47af3637e4a4065a3e93680011f3/68747470733a2f2f66696c65732e736c61636b2e636f6d2f66696c65732d7072692f5430324c50365545352d46303151504652453939412f696d6167655f66726f6d5f696f732e706e67>
> The second issue I found was that you can see the tabs from the main pages along the side of the menu screen. This is also followed by some text appearing at the bottom of the screen occasionally - this almost makes the screen look like it is glitching. Seen in the Screen shot below.
> 
>  <https://camo.githubusercontent.com/c76faefe56212a7f6afc7942349949b3b1a649dc0607c08cd12104d07070415e/68747470733a2f2f66696c65732e736c61636b2e636f6d2f66696c65732d7072692f5430324c50365545352d46303151453841315945392f696d6167655f66726f6d5f696f732e706e67>
> Lastly, The menu icon covers up parts of the text when scrolling down pages which can become annoying to users of the site.
> Seen in the screen shot below.
> 
>  <https://camo.githubusercontent.com/7acb118e46805160cb3cbcf10510ac2bc0c1a41a76508a5b45af6d4e1845da9c/68747470733a2f2f66696c65732e736c61636b2e636f6d2f66696c65732d7072692f5430324c50365545352d4630315150474e475741592f696d6167655f66726f6d5f696f732e706e67>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/evolvedbinary/eb-website/pull/8#issuecomment-792769412>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AEYN4R2ZA2XQWMJP7YLORV3TCTI6FANCNFSM4WRTWJ5Q>.
>

cmccrorie commented:
@craigsteel 
Hi,
Okay no worries thank you I will  retest it when you have uploaded it. 
Thank you .

Commit:
Changes to mobile menu pure css

craigsteel commented:
Hi, I’ve pushed a new mobile menu. It is now pure CSS so no need for any js.
I have also taken out links to About, Services and exist-db-support, as these links are not to pages but rather to areas on the home page.
I have also added the links to Twitter, LinkedIn and GitHub on the mobile menu I think it is better for the user.
I have also added a 50% white banner to help with visibility on the scroll, especially for the Afterdark page.
Let me know if you think of anything ells that would help the usability of not just the menu but the site on the whole.

Regards Craig


> On 8 Mar 2021, at 14:00, cmccrorie <notifications@github.com> wrote:
> 
> 
> @craigsteel <https://github.com/craigsteel>
> Hi,
> Okay no worries thank you I will retest it when you have uploaded it.
> Thank you .
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/evolvedbinary/eb-website/pull/8#issuecomment-792774628>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AEYN4R3HQBQMTJMRLAAJNGTTCTKARANCNFSM4WRTWJ5Q>.
> 




Craig Steel
Creative Director
tel: +44 (0) 7866 562170 <applewebdata://FF22A08D-2BA3-4332-8A20-DD282E3CB2B9/+44%20(0)%2020%207451%203353>
email: craig@craigsteel-design.com <mailto:craig@craigsteel-design.com>
Skype craig-steel
www.craigsteel-design.com <http://www.craigsteel-design.com/>
The information in this email is confidential and may be legally privileged. It is intended solely for the addressee. Access to this email by anyone else is unauthorised. If you are not the intended recipient, any disclosure, copying, distribution or any action taken or omitted to be taken in reliance on it, is prohibited and may be unlawful. Whilst every reasonable precaution has been taken, Craig Steel Design cannot accept liability for any damage that you sustain as a result of software viruses that may be contained in, or attached to this email.

 <http://www.craigsteel-design.com/>

cmccrorie commented:
Hi @craigsteel 
The last changes look great thank you, I have tested it and all working great. However If at all possible on some of the screens at the bottom for the mobile site there is a black menu that I feel does not need to be there now with how the new menu works and just looks out of place. Could this be taken off the mobile site at all? 
Thank you for your time, I have attached a screenshot of what I am on about.   
![Image from iOS](https://user-images.githubusercontent.com/68058507/110811117-4fb9d100-827e-11eb-8717-ba97050396e4.png)
Kind Regards
Callum

craigsteel commented:
Hi Callum, it shouldn’t be a problem taking out the menu in the footer, I will try and do it early next week

Regards craig



> On 11 Mar 2021, at 15:28, cmccrorie ***@***.***> wrote:
> 
> 
> Hi @craigsteel <https://github.com/craigsteel>
> The last changes look great thank you, I have tested it and all working great. However If at all possible on some of the screens for the mobile site there is a black menu that I feel does not need to be there now with how the new menu works and just looks out of place. Could this be taken off the mobile site at all?
> Thank you for your time, I have attached a screenshot of what I am on about.
>  <https://user-images.githubusercontent.com/68058507/110811117-4fb9d100-827e-11eb-8717-ba97050396e4.png>
> Kind Regards
> Callum
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/evolvedbinary/eb-website/pull/8#issuecomment-796819032>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AEYN4R2SCEK5OOIQLSYU4YLTDDORPANCNFSM4WRTWJ5Q>.
> 




Craig Steel
Creative Director
tel: +44 (0) 7866 562170 <applewebdata://53010C4F-7F3B-4D21-AD3C-A028895DB796/+44%20(0)%2020%207451%203353>
email: ***@***.*** ***@***.***>
Skype craig-steel
www.craigsteel-design.com <http://www.craigsteel-design.com/>
The information in this email is confidential and may be legally privileged. It is intended solely for the addressee. Access to this email by anyone else is unauthorised. If you are not the intended recipient, any disclosure, copying, distribution or any action taken or omitted to be taken in reliance on it, is prohibited and may be unlawful. Whilst every reasonable precaution has been taken, Craig Steel Design cannot accept liability for any damage that you sustain as a result of software viruses that may be contained in, or attached to this email.

 <http://www.craigsteel-design.com/>

Commit:
Take out mobile Menu in footer

cmccrorie commented:
@craigsteel  
Hi Craig
The changes that you have made all look good and work well. Thank you for sorting it all out. 
Kind Regards 
Callum
